apps/frontend/src/views/Preview/Preview.tsx

```diff
@@ -1,45 +1,59 @@
 import { useSelector } from 'react-redux';
 import { useMitt } from '@/plugins/mitt/react-mitt';
 import { selectPreviewTree } from '@/store/previewTree/previewTreeSlice';
-import type { PreviewElement } from '@libs/types/tree.type';
+import type { PreviewElementData } from '@libs/types/tree.type';
 import PreviewDroppable from '@components/Preview/PreviewDroppable/PreviewDroppable';
 import DynamicComponent from '@components/DynamicComponent/DynamicComponent';
 import PreviewComponentWrapper from '@/components/Preview/PreviewComponentWrapper/PreviewComponentWrapper';
 import PreviewStyle from './Preview.module.css';
+import { Suspense, useEffect, useState } from 'react';
 
 const Preview: React.FunctionComponent = () => {
     const previewTree = useSelector(selectPreviewTree);
     const emitter = useMitt('preview');
+    const [showComponent, setShowComponent] = useState(true);
+    const [render, setRender] = useState(null);
```

Comment:
Don't you think the name `render` would confuse with the built-in `React` function ?